Bottom: abf1cf21460c93b268d1b431c1bf40fd330af5e6
Top:    7c29353f62354f49e3775bf83d7770dd1ffa5640
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-20 21:34:05 +0100

Refresh of add-support-for-depth-to-trian

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 60af460..52d738a 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -368,10 +368,10 @@ void
 hidgl_draw_rect (int x1, int y1, int x2, int y2)
 {
   glBegin (GL_LINE_LOOP);
-  glVertex2f (x1, y1);
-  glVertex2f (x1, y2);
-  glVertex2f (x2, y2);
-  glVertex2f (x2, y1);
+  glVertex3f (x1, y1, global_depth);
+  glVertex3f (x1, y2, global_depth);
+  glVertex3f (x2, y2, global_depth);
+  glVertex3f (x2, y1, global_depth);
   glEnd ();
 }
